features/step_definitions/basic_steps.rb

```diff
@@ -12,6 +18,11 @@
     click_on link
 end
 
+When("I choose {string} from {string}") do |option, select_box|
+    select_box = 'dish[category_id]'
```

Comment:
You pass data into this step and then override it, either use the data you pass in or change the step.

Reply:
Done